course/templates/course/index.html

```diff
@@ -13,63 +13,63 @@ <h1>{{ welcome_text|safe }}</h1>
 {% endif %}
 
 {% if is_logged_in %}
-	<div class="frontpage-panel">
-		<h2 class="panel-heading">{% trans "My courses" %}</h2>
+	<section class="frontpage frontpage-section">
 		{% if my_instances %}
-			<section class="frontpage frontpage-section">
-				{% include "course/_course_cards.html" with instances=my_instances %}
-			</section>
+				<div class="section-heading">
+					<h2>{% trans "My courses" %}</h2>
+				</div>
+				<div class="cards">
+					{% include "course/_course_cards.html" with instances=my_instances condensed=False %}
+				</div>
 		{% else %}
 			<div class="panel-body">
 				<p class="panel-default-text">
 					{% trans "You do not currently have any active courses." %}
 				</p>
 			</div>
 		{% endif %}
-	</div>
+	</section>
 {% endif %}
 
-<div class="frontpage-panel">
-	<h2 class="panel-heading">{% trans "All ongoing courses" %}</h2>
 	<section class="frontpage frontpage-section">
-		{% site_advert as advert %}
-		{% if advert %}
-			<div class="frontpage advert-card card">
-				<a href="{{ advert.href }}" target="_blank">
-					<div class="advert-image" aria-hidden="true">
-						{% if advert.image %}
-							<img class="card-img-top" src="{{ advert.image }}" alt="" />
-						{% else %}
-							<div class="card-img-top">
-								<i class="glyphicon glyphicon-comment"></i>
-							</div>
-						{% endif %}
-					</div>
-					<div class="card-body">
-						<h3 class="card-title">{{ advert.title }}</h3>
-						<p class="card-text">{{ advert.text|safe }}</p>
-					</div>
-				</a>
+		<div class="section-heading">
+			<h2>{% trans "All ongoing courses" %}</h2>
+			<div class="action">
+				{% url 'archive' as url %}
+				<a href="{{ url }}">{% trans "All courses" %}</a>
```

Comment:
And a tiny thing, but should the link be in line with the width of the card-section? I think the panel heading -line was overflowing before a bit. It didn't matter as much when there wasn't a link, but now it seems just maybe a tad off? 

![image](https://user-images.githubusercontent.com/43036333/88372490-a8476700-cd9e-11ea-9551-97013f5d7e30.png)

Reply:
I don't think this is overflowing, it's aligned with the `.cards` container (outlined in red here)
![Näyttökuva 2020-7-24 kello 11 20 35](https://user-images.githubusercontent.com/6674046/88373031-c2ce1000-cd9f-11ea-958d-ec734a95ac5f.png)

The problem is that we show cards which don't stretch to fill the available space, and have a fixed width of `30%`, which naturally leaves some space. I'm not sure of the right way to address this, as the window size can be anything - @saarasat what are your thoughts?

Reply:
Hmm.. the flexible card-sizes do present a problem with that alignment, that's true. I can't think of a way to align it better either, maybe it can be as it is.

Reply:
Did you remove the word "see" from the link text on purpose? I thought it was helpful and suits the link well, maybe that could be added back? Then I think we are all set on my behalf :)

Reply:
I never added it, actually. It was only in my quick devtools mockup, then I forgot about it. 

But I think it makes sense, so I'll add it now.